README.md

```diff
@@ -1,12 +1,16 @@
-# PHP-dev
+# PHP-DEV
```

Comment:
why does that need to be changed?

Reply:
Thought sounds so right. I can change that back, too.